qsirecon/cli/recon_plot.py

```diff
@@ -101,6 +101,7 @@ def recon_plot():
     parser.add_argument("--padding", type=int, default=10, help="number of slices to plot")
     opts = parser.parse_args()
 
+    print(f"TMPDIR={os.getenv('TMPDIR')}")
```

Comment:
Is this environment variable getting used by xvfb?

Reply:
yes, that's where it's going to give you a directory on [gettempdir()](https://docs.python.org/3/library/tempfile.html#tempfile.gettempdir)